src/Products/Common/Resources/Resources.cs

```diff
@@ -28,10 +29,14 @@ public static string GetFreeFileName(string directory, string fileName)
                     int number = i + 1;
                     string newFileName = Path.GetFileNameWithoutExtension(fileName) + "-Copy(" + number + ")" + Path.GetExtension(fileName);
                     resultFileName = Path.Combine(directory, newFileName);
-                    if (!File.Exists(resultFileName))
+                    if (File.Exists(resultFileName))
+                    {
+                        continue;
```

Comment:
Codacy found an issue: [Remove this redundant jump.](https://app.codacy.com/gh/groupdocs-editor/GroupDocs.Editor-for-.NET-WebForms/pullRequest?prid=6152456)